pkg/types/tracefunnels.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import "github.com/uptrace/bun"
+
+type TraceFunnels struct {
+	bun.BaseModel `bun:"table:trace_funnel"`
+
+	TimeAuditable
+	UserAuditable
+	Identifiable
+	OrgID      string `json:"orgId" bun:"org_id,notnull"`
+	Name       string `json:"name" bun:"name,type:text,notnull"`
+	Category   string `json:"category" bun:"category,type:text,notnull"`
```

Comment:
what is category in tace funnel ?

Reply:
yeah this is not required since we are using a new table now, cleaned it up